doc/source/ref/wavelets.rst

```diff
@@ -249,3 +249,143 @@ from plain Python lists of filter coefficients and a *filter bank-like* objects.
     ...         return [dec_lo, dec_hi, rec_lo, rec_hi]
     >>> filter_bank = HaarFilterBank()
     >>> myOtherWavelet = pywt.Wavelet(name="myHaarWavelet", filter_bank=filter_bank)
+
+
+``ContinuousWavelet`` object
+------------------
+
+.. class:: ContinuousWavelet(name)
+
+  Describes properties of a continous wavelet identified by the specified wavelet *name*.
+  In order to use a built-in wavelet the *name* parameter must be a valid
+  wavelet name from the :func:`pywt.wavelist` list.
```

Comment:
@holgern.  Let me know if you are interested in making a PR regarding `wavelist`, otherwise I can put one together